tests/integration/targets/nae_compliance/tasks/main.yml

```diff
@@ -247,6 +289,29 @@
         }
   register: create_req
 
+- name: Create compliance requirement again
+  cisco.nae.nae_compliance:
+    <<: *nae_info
+    state: present
+    selector: requirement
+    form: |
+        {
+        "name": "ansible_comp_requirement",
+        "config_compliance_parameter": {
+        },
+        "epg_selector_a": "Web",
+        "epg_selector_b": "DataBase",
+        "requirement_type": "SEGMENTATION",
+        "communication_type": "MUST_NOT",
+        "is_all_traffic": false
+        }
+  register: create_req_again
+
+- name: Verify create_req_again
+  assert:
+    that:
+      - create_req_again.msg == "WARNING{{':'}} A compliance requirement with the same name already exist!!!"
```

Comment:
I don't think failing solves the idempotency. I should be able to do it again with it not have changed but not failing.